ReactAndroid/src/main/java/com/facebook/react/devsupport/JSDebuggerWebSocketClient.java

```diff
@@ -53,7 +50,7 @@
   private @Nullable JSDebuggerCallback mConnectCallback;
   private final AtomicInteger mRequestID = new AtomicInteger();
   private final ConcurrentHashMap<Integer, JSDebuggerCallback> mCallbacks =
-      new ConcurrentHashMap<>();
+    new ConcurrentHashMap<>();
 
   public void connect(String url, JSDebuggerCallback callback) {
     if (mHttpClient != null) {
```

Comment:
Yes, you're right, of course. (I was looking at `ReconnectingWebSocket` which is quite similar -- but uses a `boolean` field `mClosed` to prevent connection if it's already connected.)